solutions/observability/logs/logs-index-template-defaults.md

```diff
@@ -0,0 +1,76 @@
+---
+navigation_title: Default `logs` index template
+description: Learn what the default component templates and ingest pipeline hooks for logs do.
+applies_to:
+  stack: all
+  serverless: all
+products:
+  - id: observability
+---
+
+# Default `logs` index template
+
+Elastic applies a managed index template to data streams that follow the `logs-*-*` index pattern. This index template references a set of reusable component templates and an optional `@custom` ingest pipeline that standardizes how log data streams are mapped, indexed, and preprocessed.
+
+The managed `logs` index template has a priority of `100` and is automatically applied to any data stream following the `logs-*-*` index pattern, unless you override it with your own higher-priority index template.
```

Comment:
LogsDB is also enabled in this index template 

<img width="816" height="489" alt="image" src="https://github.com/user-attachments/assets/abe39af0-15b2-4ca1-b678-5935fc72cf5a" />